CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 Latest
 ------
-
+* Fixed `aiaprep` to return properly sized map.
```

Comment:
@wafels It will also work for full-disk images that have been resized (2048^2, 1024^2, etc.), but not submaps.